packages/composite-checkout/src/lib/payment-methods/stripe-credit-card-fields.js

```diff
@@ -312,6 +322,8 @@ function StripeCreditCardFields() {
 	};
 
 	if ( stripeLoadingError ) {
+		onEvent( { type: 'a8c_checkout_error', payload: { type: 'Stripe loading error' } } );
```

Comment:
Just a suggestion, but this could be simplified to `onEvent( { type: 'Stripe loading error' } )`, unless you want to group the events by type before recording them.

Reply:
Yes, this is intentional. I'm using a generic error event and then using the type property in the payload to distinguish them from each other. I just noticed the double type property. I can maybe think of another name for the second property.

Reply:
It’s fine!